### PR TITLE
docs: update readme and contributing files

### PR DESCRIPTION
**Pull request by gfellerph**
Updated information in the readme and contributing files according to the audience who will likely read them:

Readme: Developers using the Design System to develop their applications. They need to know where the docs are and how to install and use the package

Contributing: Developers who want to add a contribution to the Design System and need to know how to start the dev environment and need information about design descisions and architecture

### CODE REVIEW

File: packages/components-angular/projects/intranet-header/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 # Contributing to the Intranet Header
 
 ## Setup
-These contribution guidelines extend the [general contribution guidelines](../../../../CONTRIBUTING.md).
+
+These contribution guidelines extend the [general contribution guidelines](../../../../CONTRIBUTING.md). This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.2.0.
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/components-angular/projects/intranet-header/CONTRIBUTING.md:5]: Meant is the initial generation, not the currently installed version.

File: packages/components/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+```bash
+pnpm bootstrap
+pnpm components:start
+```
+
+To build the component for production, run:
+
+```bash
+pnpm --filter design-system-components run build
```
Comment (oliverschuerch) [inline packages/components/CONTRIBUTING.md:15]:
```suggestion
pnpm --filter design-system-components build
```

File: packages/styles/README.md [hunk 3]
```diff
@@ -1,62 +1,68 @@
...
 Install the styling package
+
 ```bash
   npm install @swisspost/design-system-styles
 ```
 
 Import the stylesheet into your project, e.g. `src/styles.scss`.
+
 ```scss
 @use "node_modules/@swisspost/design-system-styles/<bundle-name>.scss
```
Comment (oliverschuerch) [inline packages/styles/README.md:34]:
```suggestion
@use "@swisspost/design-system-styles/<bundle-name>.scss";
```

File: packages/styles/README.md [hunk 4]
```diff
@@ -1,62 +1,68 @@
...
+Available CSS bundles:
 
-### Testing
+- **index.css** (internet facing applications)
+- **intranet.css** (internal applications)
+- **basics.css** (atomic styles for font, buttons, lists etc.)
 
-To test the Schematics without writing any changes to your files, install `@angular-devkit/schematics-cli` globally and use the `schematics` command line tool.<br>
-That tool acts the same as the `generate` command of the Angular CLI, but also has a debug mode.
+## Migration giude
```
Comment (oliverschuerch) [inline packages/styles/README.md:50]:
```suggestion
## Migration guide
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**